package.json

```diff
@@ -14,38 +14,38 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@mantine/core": "^7.2.2",
-    "@mantine/hooks": "^7.2.2",
-    "clsx": "^2.0.0",
-    "eslint": "^8.51.0",
-    "eslint-config-next": "^14.0.1",
-    "geist": "^1.0.0",
-    "next": "^14.0.1",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "react-icons": "^4.11.0",
-    "react-select": "^5.7.7"
+    "@mantine/core": "7.2.2",
+    "@mantine/hooks": "7.2.2",
+    "clsx": "2.0.0",
+    "geist": "1.0.0",
+    "next": "14.0.1",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-icons": "4.11.0",
+    "react-select": "5.7.7"
   },
   "devDependencies": {
-    "@edgio/cli": "^7.4.2",
-    "@edgio/core": "^7.4.2",
-    "@edgio/devtools": "^7.4.2",
-    "@edgio/next": "^7.4.2",
-    "@edgio/prefetch": "^7.4.2",
-    "@edgio/react": "^7.4.2",
-    "@types/node": "^20",
-    "@types/react": "^18",
-    "@types/react-dom": "^18",
-    "autoprefixer": "^10",
-    "husky": ">=6",
-    "lint-staged": ">=10",
-    "postcss": "^8.4.31",
-    "postcss-preset-mantine": "^1.11.0",
-    "postcss-simple-vars": "^7.0.1",
+    "eslint": "8.51.0",
+    "eslint-config-next": "14.0.1",
+    "@edgio/cli": "7.4.2",
+    "@edgio/core": "7.4.2",
+    "@edgio/devtools": "7.4.2",
+    "@edgio/next": "7.4.2",
+    "@edgio/prefetch": "7.4.2",
+    "@edgio/react": "7.4.2",
+    "@types/node": "20",
+    "@types/react": "18",
+    "@types/react-dom": "18",
+    "autoprefixer": "10",
+    "husky": "8.0.3",
+    "lint-staged": "15.1.0",
+    "postcss": "8.4.31",
+    "postcss-preset-mantine": "1.11.0",
+    "postcss-simple-vars": "7.0.1",
     "prettier": "3.0.3",
-    "tailwindcss": "^3.3.5",
-    "ts-node": "^10.9.1",
-    "typescript": "^5"
```

Comment:
The best approach ( if you are not developing a JS library) is to lock the exact version of each packages. When your project grows each even small changes in your packages may cause problems.